practice/src/css/layouts/header.css

```diff
@@ -0,0 +1,166 @@
+.header-section {
+  padding: 16px 32px;
+}
+  
+.header-logo {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+}
+
+.header-name {
+  text-decoration: none;
+  font-family: var(--secondary-font);
+  font-weight: var(--font-weight-regular);
+  line-height: 20px;
+  color: var(--secondary-text-color);
+  font-size: var(--font-size-xsmall);
+}
+  
+.header-nav {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  gap: 10px;
+}
+  
+label {
+  cursor: pointer;
+}
+  
+#header-dropdown {
+  display: none;
+}
+  
+.nav-list {
+  display: none;
+}
+  
+#header-dropdown:checked + .nav-list {
+  grid-column: span 2;
+  display: grid;
+  gap: 15px;
+  text-align: center;
+  background: rgba(70, 107, 243, 0.1);
+  border-radius: 12px;
+}
+  
+.header-button {
+  display: none;
+}
+
+.btn-login {
+  background: rgba(70, 107, 243, 0.1);
+  border: none;
+  height: 48px;
+  width: 143px;
+  font-weight: var(--font-weight-medium);
+  font-size: var(--font-size-xsmall);
+  border-radius: 12px;
+  color: var(--secondary-text-color);
+
+  &:hover {
+    scale: 1.05;
+  }
+}
+  
+.nav-item {
+  margin-right: 50px;
+  list-style: none;
+}
+  
+.nav-link {
+  text-decoration: none;
+  color: var(--tertiary-text-color);
+  font-weight: var(--font-weight-medium);
+  font-size: var(--font-size-xxsmall);
+  font-family: var(--primary-font);
+
+  &:hover {
+    color: var(--secondary-text-color);
+  }
+}
+  
+@media only screen and (min-width: 768px) {
+  .header-section {
+    padding: 32px 64px;
+  }
+    
+  .header-button {
+    display: block;
+  }
+    
+  .header-nav {
+    grid-template-columns: 1.5fr 2fr auto;
+    gap: 10px;
+    display: grid;
+    align-items: center;
+  }
+    
+  #header-dropdown:checked + .nav-list {
+    grid-area: 2/1/3/1;
+    grid-column: span 3;
+    display: grid;
+  }
+    
+  .header-logo {
+    display: flex;
+    order: 1;
+  }
+    
+  .header-button {
+    display: flex;
+    order: 2;
+    gap: 23px;
+  }
+
+  .nav-link {
+    font-size: var(--font-size-xxsmall);
+  }
+}
+  
+@media (min-width: 1536px) {
+  .header-section {
+    padding: 30px 100px;
+  }
+  
+  .header-nav {
+  grid-template-columns: 0.5fr 1fr auto;
+  gap: 10px;
+  display: grid;
+  align-items: center;
```

Comment:
Same here, you don't need to override here. Please help me double check other places